src/dmd/mtype.d

```diff
@@ -6521,3 +6459,80 @@ const(char*)[2] toAutoQualChars(Type t1, Type t2)
     }
     return [s1, s2];
 }
+
+
+/**
+ * For each active modifier (MODFlags.const_, MODFlags.immutable_, etc) call `fp` with a
+ * void* for the work param and a string representation of the attribute.
+ */
+final int modifiersApply(TypeFunction tf, void* param, int function(void*, const(char)[]) fp)
```

Comment:
* Look at this on my phone, if this is a free function it should not be `final`
* Missing standard Ddoc sections